packages/js/src/core/sign-in.ts

```diff
@@ -34,9 +34,9 @@ export type SignInUriParameters = {
    */
   firstScreen?: FirstScreen;
   /**
-   * Specifies identifiers used in the identifier sign-in or identifier register page.
+   * Specifies identifiers used in the identifier sign-in, register, and reset password pages.
    *
-   * Available values: `email`, `phone`, `username`.
+   * Available values: `email`, `phone`, or `username`.
```

Comment:
No `or`. since they can be used together.

Reply:
OK